### DISCUSSION TITLE
Close Rofi on other key(s) than Esc?

### DISCUSSION THREAD

**Discussion by humanplayer2** [davatorium/rofi#1575] [Q&A] [answered]
I launch Rofi by `Shift-F12` (which I send by pressing `Super` using some remapping).

I would like to dismiss Rofi again by re-pressing the same, so analogously to `Esc`.

Is it possible to configure Rofi thus?

**Comment by DaveDavenport** [comment, marked as answer]
Yes. See https://github.com/davatorium/rofi/blob/next/doc/rofi-keys.5.markdown

**Reply by humanplayer2** [reply]
Thank you, I'm sorry, I had totally missed that resource. Now noticing the relevant lines in the config makes me feel quite silly. Again, thank you!
